concourse/update-content/script.bash

```diff
@@ -16,30 +16,25 @@ yarn
 approved_books_default_branch=$(curl -s https://api.github.com/repos/openstax/content-manager-approved-books | jq -r .default_branch)
 rex_default_branch=$(curl -s https://api.github.com/repos/openstax/rex-web | jq -r .default_branch)
 
-approved_books=$(curl -sL "https://github.com/openstax/content-manager-approved-books/raw/$approved_books_default_branch/approved-books.json" | jq 'map(select(.tutor_only == false)) | map(select(.server == "cnx.org"))')
-book_ids=$(jq -r 'map(.uuid) | unique | .[]' <<< "$approved_books")
+data=$(curl -sL "https://github.com/openstax/content-manager-approved-books/raw/$approved_books_default_branch/approved-book-list.json")
+
+# script will return a JSON object with book ids and new versions only for books that doesn't mach current config
+book_ids_and_versions=$(node script/entry.js transform-approved-books-data --data "$data")
+book_ids=$(echo "$book_ids_and_versions" | jq -r 'keys | .[]')
```

Comment:
if you use a -c flag you could use `entries` or something here and iterate over each json object, rather than just the key and having to look it up in the big response each time